src/ViewModels/CkanGroupViewModel.js

```diff
@@ -76,6 +77,14 @@ var CkanGroupViewModel = function(application) {
      */
     this.minimumMaxScaleDenominator = undefined;
 
+    /**
+     * Gets or sets any extra wms parameters that should be added to the wms query urls in this CKAN group.
+     * If this property is undefined or if {@link CkanGroupViewModel#filterByWmsGetCapabilities} then no parameters are added
+     * This property is observable.
+     * @type {Object}
+     */
+    this.wmsParameters = undefined;
+
     knockout.track(this, ['url', 'dataCustodian', 'filterQuery', 'blacklist']);
```

Comment:
`wmsParameters` should be added to the array here so that that property is actually observable, as its documentation claims.